updates/testing.json

```diff
@@ -19,6 +19,16 @@
           "reason": "https://github.com/coreos/fedora-coreos-streams/issues/30"
         }
       }
+    },
+    {
+      "version": "32.20200615.2.0",
+      "metadata": {
+        "rollout": {
+          "start_epoch": 1592335800,
+          "start_percentage": 0.0,
+          "duration_minutes": 4320
```

Comment:
Is there a reason for this? Why not the usual 24h?

Reply:
we used 72 hours duration for stable. So, made it 72 hours here as well.

Reply:
moved back to 48 hours window